### ISSUE TITLE
Text on media video pause button not keyboard accessible

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#5490]
## Problem/Motivation
Email sent to DRC regarding lack of accessibility.

### Describe the bug
On the text on media paragraph type, there is a pause video button when videos are uploaded as media. This button is not accessible via keyboard.

### To Reproduce

1. Go to https://www.arizona.edu/
2. Keyboard navigate through the page
3. Observe tabbing past the Pause video button

### Expected behavior

I expect to use the tab key on my keyboard and get to the Pause video button. Then be able to click Enter on my keyboard to activate the button to pause the video.

## Additional context
The Issue: The video starts playing automatically, and the “Pause video” control is not accessible via keyboard navigation. (violating WCAG 2.1.1 and 2.2.2).

The Impact: Keyboard users are unable to stop the video playback, which may cause distraction and interfere with accessibility.

<img width="337" height="236" alt="Image" src="https://github.com/user-attachments/assets/8cee32ea-a88e-47d3-8bbf-0fbc49eb1852" />
